doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -185,194 +166,157 @@ Newly opened modules and sections inherit the current settings.
    arguments ``-set`` and ``-unset`` for setting flags and options
    (cf. :ref:`command-line-options`).
 
-.. _requests-to-the-environment:
-
-Requests to the environment
--------------------------------
+Query commands
+--------------
 
-.. cmd:: Check @term
+Unlike other commands, query commands may be prefixed with
+a goal selector (:n:`@num:`) to specify which goal context, it applies to.
+If no selector is provided,
+the command applies to the current goal.  If no proof is open, then the command only applies
+to accessible objects.  (see Section :ref:`invocation-of-tactics`).
 
-   This command displays the type of :n:`@term`. When called in proof mode, the
-   term is checked in the local context of the current subgoal.
+.. cmd:: About @smart_qualid {? @univ_name_list }
 
-   .. cmdv:: @selector: Check @term
+   Displays information about the :n:`@smart_qualid` object, which,
+   if a proof is open,  may be a hypothesis of the selected goal(s),
+   or an accessible theorem, axiom, etc.:
+   its kind (module, constant, assumption, inductive,
+   constructor, abbreviation, …), long name, type, implicit arguments and
+   argument scopes. It does not print the body of definitions or proofs.
 
-      This variant specifies on which subgoal to perform typing
-      (see Section :ref:`invocation-of-tactics`).
+.. cmd:: Check @term
 
+   Displays the type of :n:`@term`. When called in proof mode, the
+   term is checked in the local context of the selected goal(s).
 
 .. cmd:: Eval @red_expr in @term
 
-   This command performs the specified reduction on :n:`@term`, and displays
+   Performs the specified reduction on :n:`@term`, and displays
    the resulting term with its type. The term to be reduced may depend on
-   hypothesis introduced in the first subgoal (if a proof is in
+   hypotheses introduced in the selected goals (if a proof is in
    progress).
 
    .. seealso:: Section :ref:`performingcomputations`.
 
 
 .. cmd:: Compute @term
 
-   This command performs a call-by-value evaluation of term by using the
-   bytecode-based virtual machine. It is a shortcut for ``Eval vm_compute in``
-   :n:`@term`.
+   Evaluates :n:`@term` using the bytecode-based virtual machine.
+   If a proof is open, :n:`@term` may reference hypotheses of the
+   selected goal.  It is a shortcut for :cmd:`Eval` :n:`vm_compute in
```

Comment:
This sentence is clearer than the equivalent one in the description of `Eval` and should probably only be present in the first one, since this one is explained as "a shortcut".